bfasst/flows.py

```diff
@@ -134,6 +134,11 @@ def flow_ic2_lse_conformal(design, flow_args, build_dir):
     design.compare_golden_files_paths.append(design.path / design.top_file)
     design.compare_golden_files_paths.extend([design.path / f for f in design.get_support_files()])
     design.golden_is_verilog = design.top_is_verilog()
+
+    flow_args[
+        ToolType.CMP
+    ] += f" {str(design.netlist_path)} {str(design.reversed_netlist_path.name)}"
```

Comment:
I don't really like using flow_args for this as these are required arguments to run the comparison tool and are not optional.  Can we just make them new arguments to the base compare tool class?

Reply:
@Tscook17

Reply:
@jgoeders  I did that originally, but I believe we ran into pylint errors with how many arguments we were adding to the function calls. There's probably some work around out there I could use, but in our meeting last Tuesday it sounded like you and Reilly wanted to use the flow_args.

Reply:
@jgoeders Sure thing.